cpg_workflows/stages/cram_qc.py

```diff
@@ -246,9 +250,10 @@ def queue_jobs(self, dataset: Dataset, inputs: StageInput) -> StageOutput | None
 
 
 def _update_meta(output_path: str) -> dict[str, Any]:
-    from cloudpathlib import CloudPath
     import json
 
+    from cloudpathlib import CloudPath
```

Comment:
Can we add a comment about why we import in the function?

Reply:
Happy to open a ticket to add some more comments for this but I feel its outside the scope of this PR :)